packages/lesswrong/lib/types/collectionTypes.ts

```diff
@@ -261,8 +261,8 @@ interface DbObject extends HasIdType {
   schemaVersion: number
 }
 
-interface HasSlugType extends DbObject {
-  slug: string | null
+interface HasSlugType {
+  slug: string
```

Comment:
The `DbObject` and `| null` cases weren't used, so I changed it to be more like `HasIdType`:
```typescript
interface HasIdType {
  _id: string
}
```